### PR TITLE
adaptor-docs fixes

### PR DESCRIPTION
**Pull request by josephjclark**
Several fixes to adaptor docs.

Note that this is branched off #505 and so is targeted at that (for now)

* [x] #503 Render examples with captions
* [x] Fix  some strange behaviour with add (adding multple items should work better now)
* [x]  #502 Render markdown
* [x] Handle errors a bit better (especially with @latest)

## Error handling

if you set an adaptor to `latest` and go to the editor, the docs panel will fail to load. Well, actually, it'll just say "loading" forever.

That issue will be fixed separately. But for now you'll at least see an error message should something go wrong.

image

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If needed, I've updated the changelog
- [ ] Amber has QA'd this feature

### CODE REVIEW

**Comment by josephjclark** [comment]
I've added `marked` (an extra 100kb of js to the pile) so that we can parse markdown on the fly. This only added a few minutes and makes a small improvement to the presentation of docs.

This is reasonably safe since the documentation comes out of code we control. Even so, I think we should be sanitising and parsing in adaptor-docs really. Maybe I should go off and do that instead? Or raise a ticket to do it soon.

File: assets/package.json [hunk 1]
```diff
@@ -11,12 +11,13 @@
   "license": "ISC",
   "dependencies": {
     "@monaco-editor/react": "^4.4.5",
-    "@openfn/describe-package": "^0.0.10",
+    "@openfn/describe-package": "file:../../kit/dist/openfn-describe-package-0.0.11.tgz",
```
Comment (josephjclark) [inline assets/package.json:14]: This is temporary and will be reverted, but I kinda had to check it in...

**Review by stuartc — APPROVED** [review]
All good here, changes are nice and clean - thanks!

**Comment by stuartc** [comment]
@amberrignell when you're done reviewing this, can you delete the branch please. 
unless @josephjclark, would it be easier to review in the other branch?

**Comment by josephjclark** [comment]
@stuartc Yeah, let's delete this and ask @amberrignell  to do some QA on the other branch and merge when she's happy. I'll sort that out.

**Pull request merged**